src/test/java/com/galvanize/customer/services/CustomerServiceTests.java

```diff
@@ -17,28 +23,51 @@
     @Autowired
     CustomerService service;
 
+    Customer c;
+
     @Before
     public void setUp() throws Exception {
-
+        c = new Customer();
+        c.setCustomerId(312L);
+        c.setFirstName("Jim");
+        c.setLastName("Jones");
+        c.setAddress("111 Main St");
+        c.setCity("Dallas");
+        c.setState("TX");
+        c.setZip("90210");
+        c.setPhoneNumber("111-222-3333");
     }
 
     @Test
     public void addCustomer() throws Exception {
+        service.addCustomer(c);
+        List<Customer> list = service.getAllCustomers();
+        assertTrue(list.size() > 0);
     }
 
     @Test
     public void getCustomersByState() throws Exception {
+        List<Customer> list = service.getCustomersByState("TX");
+        assertTrue(list.size() > 0);
+        assertEquals("Jim", list.get(0).getFirstName());
     }
 
     @Test
     public void getAllCustomers() throws Exception {
+        List<Customer> list = service.getAllCustomers();
+        assertTrue(list.size() > 0);
     }
 
     @Test
     public void getCustomerById() throws Exception {
+        Customer c1 = service.getCustomerById(312L);
```

Comment:
If @Setup is fixed, you can replace 312L with c.getCustomerId(), then test will pass.